### ISSUE TITLE
RFTools Screen Support   

### ISSUE THREAD

**Issue by Arnvior1** [mekanism/Mekanism-Feature-Requests#186]

**Comment by pupnewfster** [comment]
Going to need more information than that to even know what you are suggesting....

**Comment by Arnvior1** [comment]
For Example the energy module which shows the ammount of energy in a machine or the induction matrix on the screen

**Comment by SorinMarkov01** [comment]
Yes, this feature would be useful, compatibility with RFTools mod (curently beta 1.18) or another mod that can do something similar (like Practical Logistics 2, but this mod isn't getting updated now). It would be great, if Laser Amplifier and other machines or multiblock structures can display informations (inventory, energy, fluid, machine information), for example, via Screen in RFTools mod or better Mekanism could get its own Screen item.
Thank you in advance, for your time and respond.
